src/pages/BlogPost.js

```diff
@@ -0,0 +1,57 @@
+import { useNavigate, useParams} from "react-router-dom";
+import React from "react";
+import posts from '../data/posts.json'
+import Button from "../components/Button";
+import "../App.css"
+
+export default function BlogPost({isAuthenticatedAtr}) {
+
+    let { id } = useParams();
+    const navigate = useNavigate();
+    let postFound = posts.find( (post) => {
+        return post.id === id});
+
+    function disabled(nextOrPrev){
+        if(nextOrPrev === "+" ) {
+            return parseInt(postFound.id) === posts.length;
+        }
+        if(nextOrPrev === "-") {
```

Comment:
deze if zou eigenlijk een else, moeten zijn. dan hoef je alleen maar te checken of het + is ipv 2x een if statement